avm/ptn/ai-platform/baseline/main.bicep

```diff
@@ -38,6 +38,41 @@ param applicationInsightsSettings applicationInsightsSettingType
 @description('Optional. Settings for the AI Studio workspace hub.')
 param workspaceHubSettings workspaceHubSettingType
 
+@description('Optional. Settings for the virtual network.')
+param virtualNetworkSettings virtualNetworkSettingType
+
+@description('Optional. Settings for the Azure Bastion host.')
+param bastionSettings bastionSettingType
+
+@description('Optional. Settings for the virtual machine.')
+param virtualMachineSettings virtualMachineSettingType
+
+// ============== //
+// Variables      //
+// ============== //
+
+var createVirtualNetwork = virtualNetworkSettings.?enabled != false
```

Comment:
Curious question: Instead of an extra `enabled` flag, why not apply the logic: If the user provided e.g., the `virtualNetworkSettings` then you assume they want to deploy it?

Also (it may be a bit late though, but I wanted to mention it regardless): I wonder if `*Configuration` might be a better term than `*Settings` for the parameters. Just a thought :)

Reply:
Hi @AlexanderSehr , the reason for the enabled flag is because I want the default behaviour for the module to include the VNet. So if no VNet settings are supplied, I still want the VNet deployed

Setting `virtualNetworkSettings.enabled` to `true`, would be redundant, but the flag is still necessary for those who do not need the VNet. Same for the other `enabled` flags.

As for configuration instead of settings, will happily change :)

Reply:
Renamed settings to configuration

Reply:
@AlexanderSehr is this ok?